scripts/python/mps/scrapeLoksabhaMPs.py

```
@@ -0,0 +1,481 @@
+                "sansad_facebook": facebook,
+                "sansad_twitter": twitter,
+                "sansad_instagram": instagram,
+                "sansad_linkedin": linkedin,
+                "sansad_photo_url": photo_url,
+                "sansad_loksabha_terms": loksabha_expr,
+                "sansad_position": "",
+                "sansad_position_detail": ""
+            }
+            # Enrich with minister info
+            if ministers_map and member_id in ministers_map:
+                rec["sansad_position"] = ministers_map[member_id]["position"]
```

Ruff (0.14.3)

Undefined name `rec`

(F821)